pkg/controllers/iaa/controller.go

```diff
@@ -219,9 +219,12 @@ func provisioningUpdate(ds *apps.DaemonSet, dp *devicepluginv1.IaaDevicePlugin)
 	found := false
 
 	for _, container := range ds.Spec.Template.Spec.InitContainers {
-		if container.Name == "intel-iaa-initcontainer" && container.Image != dp.Spec.InitImage {
+		if container.Name == "intel-iaa-initcontainer" {
```

Comment:
our code has this minor inconsistency that the `const` is not used all over the place (also, it has a typo?) can you check other places too:
```suggestion
		if container.Name == initcontainerName {
```

Reply:
Added more `const`s and fixed the typo.